scripts/renderers.py

```diff
@@ -20,31 +20,65 @@ class CsvRenderer():
     Used to output toggles+annotations data as CSS
     """
 
-    def transform_toggle_data_for_csv(self, envs_data):
+    def render_csv_report(self, idas, file_path="report.csv", toggle_types=None, header=None, summarize=False):
         """
-        Retrieve list of individual toggle datums from envs_data
+        takes data, processes it, and outputs it in csv form
+        """
+        if summarize:
+            toggle_info_structured_dicts = self.summarize_data(idas)
+        else:
+            toggle_info_structured_dicts = self.output_full_data(idas)
+
+        toggles_info_flattened_dicts = self.add_info_source_to_dict_keys(toggle_info_structured_dicts)
+
+        sorted_toggles_dicts = self.filter_and_sort_toggles(toggles_info_flattened_dicts, toggle_types)
+        header = self.get_sorted_headers_from_toggles(sorted_toggles_dicts, header)
+        self.write_csv(file_path, sorted_toggles_dicts, header)
+
+    def output_full_data(self, idas):
+        """
+        return full data about toggles
+        Essentially, one row per toggle state(one toggle with multiple states will have multilple rows)
```

Comment:
Note: fixes spelling typo (and changes formatting).
```suggestion
        Return full data about toggles.

        Essentially, one row per toggle state (one toggle with multiple states will have multiple rows)
```